src/containers/shared/types.ts

```diff
@@ -21,33 +24,14 @@ export type ExplorerAmount = {
   amount: number
 }
 
-export interface Node {
-  DeletedNode?: any
-  ModifiedNode?: any
-  CreatedNode?: any
-  LedgerEntryType: string
-}
-
-export interface Meta {
-  AffectedNodes: Node[]
-}
-
-export interface Memo {
-  Memo: {
-    MemoType: string
-    MemoData: string
-    MemoFormat: string
-  }
-}
-
 export interface Tx {
```

Comment:
Is this type needed anymore? Seems like it could be replaced by the xrpl.js `Transaction`